src/snapshot/prepare_snapshot_received_proposals_airtable.js

```diff
@@ -51,45 +51,16 @@ const prepareProposalsForSnapshot = async (curRound) => {
 
   await Promise.all(
     proposals.map(async (proposal) => {
-      getProposalRecord(proposal, proposals)
-      try {
-        const wallet_0x = proposal.get('Wallet Address')
-        const proposalStanding = proposal.get('Proposal Standing')
-
-        // Please update enums as required
-        const goodStanding =
-          proposalStanding === Standings.Completed ||
-          proposalStanding === Standings.Refunded ||
-          proposalStanding === Standings.Undefined ||
-          proposalStanding === Standings.Unreported ||
-          proposalStanding === Standings.NewProject
-
-        if (hasEnoughOceans(wallet_0x) && goodStanding === true) {
```

Comment:
If a proposal initially didn't have the 500 oceans required and was therefore rejected, but then adds the 500 oceans, will its state be updated according to the new logic?

Reply:
This logic works when creating the Snapshot proposal. We don't need to check the 500 Oceans required here because Daobot checks that every minute and will set the proposal to `Rejected` state if the wallet doesn't have enough Ocean tokens.

Reply:
Awesome, thanks for taking the time and explain this to me. You are right, this logic is run only when the proposal is created and everything should be set by then.